src/components/conjunction/ConjunctionManoeuvreSupport.tsx

```diff
@@ -1,10 +1,25 @@
-import { useTranslations } from 'next-intl';
+import { getTranslations } from 'next-intl/server';
 
-const ConjunctionManoeuvreSupport = () => {
-  const t = useTranslations('Accordions.Conjunction.mtp_chart');
+import type { TypeManoeuvrePlotWithUserMetadataOut } from '@/__generated__/data-contracts';
+import { getManoeuvrePlot } from '@/actions/getManoeuvrePlot';
+import { getManoeuvrePlotsManoeuvrePlotId } from '@/actions/getManoeuvrePlotsManoeuvrePlotId';
+
+import MtpChart from '../charts/mtp-chart/MtpChart';
+
+type ConjunctionManoeuvreSupportProps = {
+  plot: TypeManoeuvrePlotWithUserMetadataOut;
+};
+
+const ConjunctionManoeuvreSupport = async ({ plot }: ConjunctionManoeuvreSupportProps) => {
+  const t = await getTranslations('Accordions.Conjunction.mtp_chart');
+
+  const { data } = await getManoeuvrePlotsManoeuvrePlotId(plot?.id);
+  const manoeuvrePlot = await getManoeuvrePlot(data.presignedUrl);
 
   return (
-    <div data-pdf={t('title')}></div>
+    <div data-pdf={t('title')}>
```

Comment:
can't you just extend MtpChartsProps with HTML props and pass data-pdf to component?

Reply:
Fixed